### ISSUE TITLE
Language pack German

### ISSUE THREAD

**Issue by sirrooster** [Feodor2/Mypal#338]
German language pack (mypal-lang-de-29.0.0.xpi) does not install.

**Comment by Feodor2** [comment]
No, it does, so provide more detailes what error do you have, how to reproduce and else...
![image](https://user-images.githubusercontent.com/19492771/108538521-8f2c7780-72e7-11eb-938b-4c65d2690486.png)

**Comment by AroKol78** [comment]
**about:config**  --> `general.useragent.locale`  `de`
